### ISSUE TITLE
Set up mkdocs site in PD

### ISSUE THREAD

**Issue by fyliu** [hackforla/peopledepot#166]
### Overview

We need to convert our docs to use the mkdocs documentation generator. It will make it easier to work on documentation. This issue should be done as simply as possible without adding the bells and whistles.

### Action Items

- [x] use mkdocs from hackforla project
- [x] customize setting to PD
- [x] set up gh-workflow to update gh-pages on git push
- [x] write documentation on how we set up and use mkdocs
- [ ] release #167 from ice box
- [ ] check and release #164 from ice box
- [ ] take #182 out of the ice box if this is the last dependency

### Resources/Instructions

- References for documentation structure;
  - [old PD sphinx site](https://fyliu.github.io/peopledepot-sphinx/tools/git.html)
  - [black docs](https://black.readthedocs.io/en/stable/contributing/reference/reference_summary.html)
- [website-wiki's resources on setting up mkdocs](https://github.com/hackforla/website-wiki)
- Vanessa Rodriguez (gh: vraer) is the person working on the website's wiki mkdocs site. We can ask her about mkdocs.

**Comment by fyliu** [comment]
I've actually been working on this one and I created a docker config to build an image so our devs won't have to install mkdocs on their own machines. But I realized it can work for all the other teams using mkdocs at hfla, so I went to set up the [docker-mkdocs](https://github.com/hackforla/docker-mkdocs) repo for that purpose. I'm planning to finish that soon and then make our setup just download that image rather than build it ourselves.

This should be very fast once I come back to it.
